doc/README_Sink.md

```diff
@@ -168,19 +168,23 @@ To delete the created Azure Cosmos DB service and its resource group using Azure
+| :--- |:--------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------| :--- |
+| topics | list    | A list of Kafka topics to watch                                                                                                                              | Required |
+| connector.class | string  | Classname of the Cosmos DB sink. Should be set to `com.azure.cosmos.kafka.connect.sink.CosmosDBSinkConnector`                                                | Required |
+| connect.cosmos.connection.endpoint | uri     | Cosmos endpoint URI string                                                                                                                                   | Required |
+| connect.cosmos.master.key | string  | The Cosmos primary key that the sink connects with                                                                                                           | Required |
+| connect.cosmos.databasename | string  | The name of the Cosmos database the sink writes to                                                                                                           | Required |
+| connect.cosmos.containers.topicmap | string  | Mapping between Kafka Topics and Cosmos Containers, formatted using CSV as shown: `topic#container,topic2#container2`                                        | Required |
+| connect.cosmos.connection.gateway.enabled | boolean | Flag to indicate whether to use gateway mode. By default it is `false`.                                                                                      | Optional  |
+| connect.cosmos.sink.bulk.enabled | boolean | Flag to indicate whether bulk mode is enabled. By default it is `true`.                                                                                      | Optional |
+| connect.cosmos.sink.maxRetryCount | int     | Max retry attempts on transient write failures. By default it is `10` times. NOTE: This is different from max throttling retry attempts, which are infinite. | Optional |
+| connect.cosmos.connection.sharing.enabled | boolean | Flag to enable connection sharing between instances of cosmos clients on the same jvm. NOTE: If you have set 'connect.cosmos.connection.gateway.enabled' to true, then this configure will not make any difference.                                                                | Optional |
+| key.converter | string  | Serialization format for the key data written into Kafka topic                                                                                               | Required |
+| value.converter | string  | Serialization format for the value data written into the Kafka topic                                                                                         | Required |
```

Comment:
@xinlian12 this looks like a new feature / fix. I don't see it in the changelog. Can you please check?

Reply:
nvm, ignore my above comment.